docs/meson.build

```diff
@@ -0,0 +1,10 @@
+
+if rstlint.found()
+  test(
```

Comment:
nit, here, generally, is 2-space-indent a meson standard? if not could we get 4-spaces to match local style?

Reply:
I don't think meson itself has a documented standard, but 2-space indent is what the Emacs meson-mode gives me

Reply:
ok, do you mind following local style (4 spaces) then please

Reply:
Have switched to 4-space